api/views.py

```diff
@@ -439,7 +439,7 @@ def upload_sample_input(request):
     data = request.data
     demo_id = data["demo_id"]
     demo = Demo.objects.get(id=demo_id)
-    for key, value in data.iteritems():
+    for key, value in data.items():
```

Comment:
Why?

Reply:
python 3 no longer supports iteritems, as I mentioned in the comment above.

Reply:
You'll need to return iter(data.items()) right? https://docs.python.org/3.3/library/stdtypes.html#dictionary-view-objects

Reply:
I think the for .. in loop implicitly uses an iterator

Reply:
:+1: